src/cdk-experimental/ui-patterns/behaviors/list-selection/list-selection.spec.ts

```diff
@@ -157,6 +157,36 @@ describe('List Selection', () => {
     });
   });
 
+  describe('#toggleOne', () => {
```

Comment:
`toggleOne`, `selectOne` are weird method names. If I understand correctly they're toggling/selecting the active item, so maybe call them `toggleActive`, `selectActive`

Reply:
`toggleOne` and `selectOne` are for ensuring only a single option remains selected in the entire list